df.art.xml

```diff
@@ -1045,14 +1045,14 @@
         <enum-item name='AccentedBeat' comment="X"/>
         <enum-item name='Beat' comment="x"/>
         <enum-item name='PrimaryAccent' comment="!"/>
-        <enum-item/>
+        <enum-item name='SilentEarly' comment="-`"/>
```

Comment:
Do you think these comments would be useful enough to expose as `enum-attr`s instead? Doing so would make them accessible programmatically.

Reply:
I don't think the comment info would be useful, actually. The only place it's used is in the description in exported Legends info. If it has any use it would probably be on a more primitive level: Looking at it (and the Legends output it comes from), you'll see there are actually two "types" at play interacting to generate the enums used:
1. Beat type: Silent, AccentedBeat, Beat, and PrimaryAccent
2. Beat timing type: Normal, Early, and Late (or Syncope, as I've called it as that's what it's commonly called in music).

I did consider a comment prior to the type listing all the 7 elements (normal beat is indicated by the absence of a timing modifier).